js/compiler.ts

```diff
@@ -405,6 +405,7 @@ export class DenoCompiler
       assert(
         mediaType === MediaType.TypeScript || mediaType === MediaType.JavaScript
       );
+      console.log("pre emit output");
```

Comment:
before shipping, these should be util's log (hidden behind debug flag).

Reply:
Of course, just needed some guidance during debugging.